<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookingWindow</class>
 <widget class="QDialog" name="BookingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>750</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding"/>
  </property>
  <property name="windowTitle">
   <string>预订车位</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_status">
     <property name="text">
      <string>加载中...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_refresh">
     <property name="text">
      <string>刷新附近停车场</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="list_lots"/>
   </item>
   <item>
    <widget class="QComboBox" name="combo_space_type">
     <property name="placeholderText">
      <string>选择车位类型</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="timeLayout">
     <item>
      <widget class="QLabel" name="label_start">
       <property name="text">
        <string>开始时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="datetime_start"/>
     </item>
     <item>
      <widget class="QLabel" name="label_end">
       <property name="text">
        <string>结束时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="datetime_end"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="btn_confirm">
       <property name="text">
        <string>确认预订</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
